src/api/libcellml/model.h

```diff
@@ -68,11 +68,19 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
     /**
      * @brief Add a child units to this model.
      *
-     * Add a copy of the given units as a child units of this model.
+     * Add the given units as a child units of this model. If the units
+     * already belong to another model, they are moved to this one, and
```

Comment:
belongs?

Reply:
Have I said how much I loathe this plural-is-singular-of-something-else naming?? Gah! In this case, since we don't flag units as `Units`, I think it should be "belong" (ie: all plain English, so all plain English grammar).  But either way it needs to change (deleting the "a" in previous sentence). The options would be:
- Add the given units as child units of this model. If the units already belong to another model, they are moved to this one, and ...
- Add the given `Units` as a child `Units` of this model. If the `Units` already belongs to another model, they are moved to this one, and ...
- Add the given units item as a child units item of this model. If the units item already belongs to another model, it is moved to this one, and ...

Reply:
I feel like your second option might be more in line with what is currently being done. So, I would go with that. (Otherwise, agreed about that units stuff...)

Reply:
d9b0052d